src/main/java/service/RamlDiffService.java

```diff
@@ -67,10 +78,35 @@ public void diff(String later, String older) throws Exception {
    * "src/test/resources/01-bookservice.raml"); }
    */
 
-  public static void main(String[] args) throws Exception {
+/*  public static void main(String[] args) throws Exception {
     new RamlDiffService().diff("src/test/resources/github.raml", "src/test/resources/github-api-v3.raml");
   }
-
+*/
+  
+  public static void main(String[] args) throws Exception {
+    get(RAML_DIFF_SERVICE, (request, response) -> { 
+        return RAML_DIFF_SERVICE_AVAILABLE_TEXT;    
+    });
+    post(FIND_ALL_DIFFERENCES_CONTEXT, (request, response) -> {
+      String diffRequestAsJson = request.body();
+      RamlDiffServiceRequest requestObj = JsonUtility.toRequestObject(diffRequestAsJson);
+      String oldRamlFilePath = requestObj.getOldRamlFileURL();
+      String newRamlFilePath = requestObj.getNewRamlFileURL();
+      
+      URL oldFileUrl = new URL(oldRamlFilePath);
+      URL newFileUrl = new URL(newRamlFilePath);
+      
+      List<ActionDiff> allDifferences = new RamlDiffService().diff(newFileUrl.getFile(), oldFileUrl.getFile());
+      
+      RamlDiffServiceResponse responseObj = new RamlDiffServiceResponse();
+      responseObj.setRamlFileDifferences(allDifferences.stream().flatMap(eachDiff -> {

```

Comment:
Added Response Object to capture any error messages in Raml Difference finders. I had not written that piece of code. Have removed the Request & Response beans as well as JSON Utility.